hw/acpi/piix4.c

```diff
@@ -344,11 +344,17 @@ static const VMStateDescription vmstate_acpi = {
     }
 };
 
+static void piix4_reset_nofw(void *opaque)
+{
+    PIIX4PMState *s = opaque;
+
+    acpi_pcihp_reset(&s->acpi_pci_hotplug);
+}
+
 static void piix4_reset(void *opaque)
```

Comment:
since you are fixing an issue with nofw, I think you should't modify this function

Reply:
Actually, the modified codes are from the patch <6ba2bfbe> which is just for nofw case.